docs/rfcs/003-pull-request-review.md

```diff
@@ -16,162 +16,156 @@ Peer review is also a critical part of the path to acceptance for pull requests
+Uncollapsed (default) | Collapsed
+--- | ---
+![files](https://user-images.githubusercontent.com/378023/46536560-d3bb4200-c8e9-11e8-9764-dca0b84245cf.png) | ![collapsed files](https://user-images.githubusercontent.com/378023/46931273-7069a680-d085-11e8-9ea7-c96a1772fe27.png)
 
-#### Filter Options
+* For large diffs, the files can be collapsed to get a better overview.
 
-<img width="731" alt="filter options" src="https://user-images.githubusercontent.com/7910250/46392373-03720c00-c696-11e8-9a1b-fe6bc6238769.png">
 
-The default is to show all files, all authors, and unresolved comments.
+### Reviews (tab)
 
-Filtering based on file type limits the diff view to displaying only that file type.
+Clicking on the "Reviews" tab shows all reviews of a pull request. This is akin to the review summaries that appear on the "Conversation" tab on dotcom. In addition, each review also includes review comments and their diff.
 
-> :construction: Consider adding a "Find" input field that allows us to filter based on search term (which could be a file name, an author, a variable name, etc).
+![reviews](https://user-images.githubusercontent.com/378023/46535563-c81a4c00-c8e6-11e8-9c0b-6ea575556101.png)
 
-Clicking an author's avatar displays only their review information.
+Uncollapsed (default) | Collapsed
+--- | ---
+![reviews](https://user-images.githubusercontent.com/378023/46535563-c81a4c00-c8e6-11e8-9c0b-6ea575556101.png) | ![collapsed reviews](https://user-images.githubusercontent.com/378023/46926357-62a72780-d06b-11e8-9344-23389d1c727c.png)
 
-Clicking "unresolved" shows only resolved comments, helping users stay focused on comments that need to be addressed.
+* Comments can be collapsed to get a better overview.
+* Reviews get sorted by "urgency". Showing reviews that still need to get adressed at the top:
+  1. "recommended" changes
+  2. "commented" changes
+  3. "no review" (when a reviewer only leaves review comments, but no summary)
+  4. "approved" changes
+  5. "previous" reviews (when a reviewer made an earlier review and it's now out-dated)
+* Within each group, sorting is done by "newest first".
 
-Clicking "resolved" shows only resolved comments. This allows users to quickly see what has already been addressed.
+#### Create a new review
 
-Checking "all comments" shows both resolved and unresolved comments.
+##### `+` Button
 
-Clicking "none" hides all comments, in the event that users want to see diff information only.
+Hovering along the gutter within a pull request diff region in a `TextEditor` or a `PullRequestDetailItem` reveals a `+` icon. Clicking the `+` icon reveals a new comment box, which may be used to submit a single comment or start a multi-comment review:
```

Comment:
We can only create new review comments at certain locations within the files, though: it has to be within a diff region on the PR. Can we add some kind of gutter decoration to signal where these regions are?